src/main/java/kitchenpos/products/tobe/domain/Product.java

```diff
@@ -0,0 +1,42 @@
+package kitchenpos.products.tobe.domain;
+
+
+import kitchenpos.products.infra.PurgomalumClient;
+
+import javax.persistence.*;
+import javax.validation.constraints.Email;
+import java.math.BigDecimal;
+import java.util.UUID;
+
+@Table(name = "product")
+@Entity
+public class Product {
+    @Column(name = "id", columnDefinition = "varbinary(16)")
+    @Id
+    private UUID id;
+
+    @Embedded
+    private ProductName name;
+
+    @Embedded
+    private ProductPrice price;
+
+    protected Product() { /* empty */ }
```

Comment:
```suggestion
    protected Product() {}
```
불필요한 주석은 없어도 될 것 같아요~